xdist/dsession.py

```diff
@@ -521,11 +94,13 @@ def pytest_collection(self):
 
     @pytest.mark.trylast
     def pytest_xdist_make_scheduler(self, config, log):
-        dist = config.getvalue("dist")
-        if dist == "load":
-            return LoadScheduling(config, log)
-        elif dist == "each":
-            return EachScheduling(config, log)
+        dist = config.getvalue('dist')
+        schedulers = {
+            'each': EachScheduling,
+            'load': LoadScheduling,
+            'loadsuite': LoadSuiteScheduling
+        }
+        return schedulers[dist](config, log)
```

Comment:
Actually it seems this is something everybody wants, so let's continue the discussion. Did you read the comments about using a marker to customize this behavior? Do you have an opinion on that?